### DISCUSSION TITLE
Media (not @media) is missing in bootstrap 5 alpha

### DISCUSSION THREAD

**Discussion by Molla143** [twbs/bootstrap#31763] [General] [answered]
Before opening:

- [Search for duplicate or closed issues](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue)
- Read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

Feature requests must include:
media

- As much detail as possible for what we should add and why it's important to Bootstrap
- Relevant links to prior art, screenshots, or live demos whenever possible

**Comment by patrickhlauke** [comment]
no idea what this even means, sorry

**Reply by patrickhlauke** [reply]
converting to discussion in case you can/want to elaborate a bit further / flesh this out

**Reply by Molla143** [reply]
Media object missing in bootsrap 5 alpha...
https://getbootstrap.com/docs/4.0/layout/media-object/

**Comment by patrickhlauke** [comment, marked as answer]
> Dropped the `.media` component as it can be built with utility classes. [See #28265](https://github.com/twbs/bootstrap/pull/28265).

https://v5.getbootstrap.com/docs/5.0/migration/#grid-and-layout

**Comment by suhailvs** [comment]
here is a simple demo of media object, taken from https://stackoverflow.com/a/64057706/2351696

    <div class="d-flex border p-3">
      <img src="https://cdn.pixabay.com/photo/2016/11/18/23/38/child-1837375_960_720.png" alt="John Doe"
         class="flex-shrink-0 mr-3 mt-3 rounded-circle" style="width:60px;height:60px;">
      <div>
        <h4>John Doe <small>Posted on February 19, 2016</small></h4>
        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua.</p>
      </div>
    </div>
